add-on/chrome/bootstrap.js

```diff
@@ -347,6 +349,41 @@ var WindowListener = {
 
         this.maybeAddCopyPanel();
         this.updateToolbarState();
+
+        this.addSidebar();
+      },
+
+      /* XXX akita make this into it's own object. */
+      addSidebar: function() {
+        let ownerDocument = gBrowser.ownerDocument;
+        var browser = ownerDocument.getElementById("browser");
+
+        let sidebarBrowser = document.createElementNS(kNSXUL, "browser");
+        sidebarBrowser.setAttribute("id", "loop-side-browser");
+        sidebarBrowser.setAttribute("disable-history", "true");
+        sidebarBrowser.setAttribute("disable-global-history", "true");
+
+        // XXX something like these seem likely to be required once we
+        // electrolyze this along with a URI_MUST_LOAD_IN_CHILD in
```

Comment:
The sidebar is already URI_MUST_LOAD_IN_CHILD. I think this probably doesn't apply, however, lets change the `XXX` to `XXX akita` and verify it before too much longer.

Actually I think having the must load in child forces it into a remote browser, but specifying the browser element remote by default optimises the process.

So just update the `XXX` for now please.